redfish-exporter/.env

```diff
@@ -10,35 +10,35 @@ SLURM_USER="slurm user here"
 SLURM_TOKEN="token string here, from secret when for real"
 SLURM_CONTROL_NODE="slurm control node IP:Port"
 
-TRIGGER_EVENTS="[ \
```

Comment:
Having indentation is good for code readability, but eventually, we should add YAML support. Complex data isn't really suitable in this format, so do as you see fit.

Reply:
Looks like the testing team couldn't run it without removing the whitespaces.